src/components/App.tsx

```diff
@@ -123,16 +129,27 @@ export const App = () => {
       ]);
 
       const completeSolarRecords = solarEvents.map(solarEvent => {
-        return {
+        const record: { [key: string]: any } = {
```

Comment:
I would use `Record` here, but the effect is the same:
```suggestion
        const record: Record<string, any> = {
```